rllib/agents/trainer.py

```diff
@@ -222,7 +222,7 @@
     # generic ModelV2 `input_dicts` that can be requested by the model to
     # contain different information on the ongoing episode.
     # NOTE: Only supported for PyTorch so far.
-    "_use_trajectory_view_api": False,
+    "_use_trajectory_view_api": "auto",
```

Comment:
Probably None is more clear.

Reply:
ok

Reply:
done.
Also fixed for SAC's `target_entropy`, which also had a default value of "auto" (but would also accept None). We should keep this unified.